package.json

```diff
@@ -1,67 +1,67 @@
-{
-	"name": "ardrive-cli",
-	"version": "0.2.12",
-	"description": "The ArDrive Command Line Interface (CLI) contains all of the needed security, file synchronization and Arweave wallet capabilities via a node.js application.",
-	"main": "./lib/index.js",
-	"bin": "./lib/index.js",
-	"types": "./lib/index.d.ts",
-	"dependencies": {
-		"@types/node": "^14.14.32",
-		"@types/uuid": "^8.3.0",
-		"ardrive-core-js": "REPLACE WITH you local ardrive-core",
-		"arweave": "1.10.11",
-		"arweave-bundles": "^1.0.3",
-		"community-js": "^1.1.36",
-		"progress": "^2.0.3",
-		"prompt-password": "^1.2.0",
-		"prompt-sync": "^4.2.0",
-		"prompts": "^2.4.0",
-		"typescript": "^4.2.3",
-		"uuid": "^8.3.2"
-	},
-	"devDependencies": {
-		"@types/jwk-to-pem": "^2.0.0",
-		"@types/prompt-sync": "^4.1.0",
-		"@types/prompts": "^2.0.9",
-		"@typescript-eslint/eslint-plugin": "^4.18.0",
-		"@typescript-eslint/parser": "^4.18.0",
-		"eslint": "^7.23.0",
-		"eslint-config-prettier": "^8.1.0",
-		"eslint-plugin-prettier": "latest",
-		"husky": "^=6",
-		"lint-staged": "^11.0.0",
-		"prettier": "^2.2.1",
-		"rimraf": "^3.0.2",
-		"ts-node": "^9.1.1",
-		"tsc-files": "^1.1.2"
-	},
-	"scripts": {
-		"clean": "rimraf lib",
-		"format": "prettier --write \"src/**/*.ts\"",
-		"lint": "eslint . --ext .ts",
-		"lintfix": "eslint . --ext .ts --fix",
-		"typecheck": "tsc --noemit",
-		"build": "yarn clean && tsc",
-		"version": "yarn run format && git add -A src",
-		"postversion": "git push && git push --tags",
-		"start": "yarn run build && node ./lib/index.js"
-	},
-	"husky": {
-		"hooks": {
-			"pre-commit": "lint-staged"
-		}
-	},
-	"license": "AGPL-3.0-or-later",
-	"author": {
-		"name": "Permanent Data Solutions Inc",
-		"email": "info@ardrive.io",
-		"website": "https://ardrive.io"
-	},
-	"repository": {
-		"type": "git",
-		"url": "https://github.com/ardriveapp/ardrive-cli.git"
-	},
-	"files": [
-		"lib/**/*"
-	]
-}
+{
+	"name": "ardrive-cli",
```

Comment:
Did your IDE impose some whitespace changes here?

Reply:
Yes, looks like it fixed the tabbing here from `2` to `4`. Seems that the .json files are skipped by the prettier format command, but when `.editorconfig` is setup my IDE formats .json files correctly